lumicks/pylake/kymotracker/kymotracker.py

```diff
@@ -107,11 +109,17 @@ def track_greedy(
         position, time, m0 = position[mask], time[mask], m0[mask]
 
     peaks = KymoPeaks(position, time, m0)
-    peaks = merge_close_peaks(peaks, np.ceil(0.5 * line_width))
+    peaks = merge_close_peaks(peaks, np.ceil(0.5 * line_width_pixels))
 
+    velocity_scale = position_scale / kymograph_channel.line_time_seconds
+    diffusion_scale = position_scale ** 2 / kymograph_channel.line_time_seconds
     lines = points_to_line_segments(
         peaks,
-        kymo_score(vel=vel, sigma=sigma, diffusion=diffusion),
+        kymo_score(
+            vel=vel / velocity_scale,
+            sigma=sigma / position_scale if sigma else 0.5 * line_width_pixels,
+            diffusion=diffusion / diffusion_scale,
+        ),
```

Comment:
I wonder if there should be a small comment here. It's not super apparent from first glance at the code what exactly is going on here. Of course one could reason it all through to get to what the units are for everything, but then I would argue it'd be helpful to just state it upfront to make our lives easier in the future

Reply:
Yeah, that's a fair point. I've moved it to its own section for clarity. The variable names should make it more clear what's happening.

```
    # Convert algorithm parameters to pixel units
    velocity_pixels = vel * kymograph_channel.line_time_seconds / position_scale
    diffusion_pixels = diffusion / (position_scale ** 2 / kymograph_channel.line_time_seconds)
    sigma_pixels = sigma / position_scale if sigma else 0.5 * line_width_pixels
```